src/domain/occurrence/enrolmentTable/additionalInfo/AdditionalInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { EnrolmentFieldsFragment } from '../../../../generated/graphql';
+import { translateValue } from '../../../../utils/translateUtils';
+import styles from './additionalInfo.module.scss';
+
+interface Props {
+  enrolment: EnrolmentFieldsFragment;
+}
+
+const AdditionalInfo: React.FC<Props> = ({ enrolment }) => {
+  const { t } = useTranslation();
+
+  const language =
+    enrolment.person?.language || enrolment.studyGroup.person.language;
+  const studyGroupPersonInfo = [
+    enrolment.studyGroup.person.phoneNumber,
+    enrolment.studyGroup.person.emailAddress,
+  ].filter((e) => e);
+  const personInfo = [
+    enrolment.person?.phoneNumber,
+    enrolment.person?.emailAddress,
+  ].filter((e) => e);
+  const extraNeeds = enrolment.studyGroup.extraNeeds;
```

Comment:
Would something like getEnrolmentField be a good fit here?

Reply:
Added helper function